module/Education/src/Education/Form/Upload.php

```diff
@@ -4,9 +4,13 @@
 
 use Zend\Form\Form;
 use Zend\InputFilter\InputFilter;
+use Zend\InputFilter\InputFilterProviderInterface;
 use Zend\I18n\Translator\TranslatorInterface as Translator;
 
-class Upload extends Form
+/**
+ * Upload a summary
+ */
+class Upload extends Form implements InputFilterProviderInterface
```

Comment:
Why not call the file/class UploadSummary instead? That makes the comment superfluous